modules/heat_conduction/doc/content/source/constraints/ADInterfaceJouleHeatingConstraint.md

```diff
@@ -12,22 +12,29 @@ As such, the `ADInterfaceJouleHeatingConstraint` takes as a required argument th
 !alert note title=Closed Gap Interface Assumed by this Class
 The `ADInterfaceJouleHeatingConstraint` class should only be employed in simulations when the user is certain that electric potential flux is nonzero only when the interface gap is closed. This class may also be used in simulations with an open gap at the interface, so long as the electric potential flux across that gap is zero while the interface gap is open.
```

Comment:
Since this has already been merged, could this usage of electrical flux here be fixed up in a follow up PR?

Reply:
Made in #22312